lighthouse-core/lib/tracehouse/trace-processor.js

```diff
@@ -579,7 +586,7 @@ class TraceProcessor {
    * @return {TraceOfTabWithoutFCP}
   */
   static computeTraceOfTab(trace, options) {
-    const {timeOriginDeterminationMethod = 'lastNavigationStart'} = options || {};
+    const {timeOriginDeterminationMethod = 'auto'} = options || {};
```

Comment:
in what situations would `'auto'` be used?

Reply:
> > in what situations would `'auto'` be used?

> all situations, it's the default :)

It could cause some difficult to debug surprises (e.g. something's not using navStart, you have to know the user's trace has a marker in it). Is the intention that most audits will just work™ doing it this way, though?